community/fvwm/APKBUILD

```diff
@@ -2,9 +2,9 @@
 # Maintainer: ScrumpyJack <scrumpyjack@st.ilet.to>
 pkgname=fvwm
 pkgver=2.6.7
-pkgrel=0
+pkgrel=1
 pkgdesc="FVWM is a powerful desktop window manager for the X  Window system"
-url="http://fvwm.org"
```

Comment:
hm, weird. When I tried the link it did not work. Maybe the website got down.

I will close this PR, thanks for reviewing it.